reaktive/src/jsCommonTest/kotlin/com/badoo/reaktive/scheduler/MainSchedulerTest.kt

```diff
@@ -9,4 +20,62 @@ class MainSchedulerTest {
         val executor = scheduler.newExecutor()
         executor.submit {}
     }
+
+    @Test
+    fun delayed_task_executed_with_delay(): AsyncTestResult {
+        val scheduler = MainScheduler()
+        val executor = scheduler.newExecutor()
+        var executed = false
+        executor.submit(200.milliseconds) { executed = true }
+
+        return observableOf(
+            100.milliseconds, 300.milliseconds
```

Comment:
Can be on one line for better formatting.

Reply:
done